### ISSUE TITLE
Add Create: Copycats+

### ISSUE THREAD

**Issue by juh9870** [EnigmaticaModpacks/Enigmatica9#992]
### URL

https://www.curseforge.com/minecraft/mc-mods/copycats

### Why would you like the mod added?

The main draw of this mod as compared to framed blocks is having proper support for connected textures, as well as adding some block shapes that are not supported by framed blocks.

**Comment by NielsPilgaard** [comment]
Looks very cool, but it's a bit late to switch from Framed Blocks I'm afraid. We'll consider it.

**Comment by juh9870** [comment]
I'm not asking to replace framed blocks, a ton of people use it. I suggest adding this as an alternative option.

**Comment by NielsPilgaard** [comment]
Fair enough. Having two mods that does nearly the exact same thing comes with a performance penalty though, so it's a trade-off.
